src/app.module.ts

```diff
@@ -4,6 +4,7 @@ import { AppService } from './app.service';
 import { ConfigModule } from '@nestjs/config';
```

Comment:
**praise**: Excellent use of the ConfigModule to manage environment variables. This is a best practice for configuration management in Nest.js applications.